src/common/KernelBase.hpp

```diff
@@ -72,7 +72,12 @@ class KernelBase
 
   void setVariantDefined(VariantID vid);
   void addVariantTuningName(VariantID vid, std::string name)
-  { variant_tuning_names[vid].emplace_back(std::move(name)); }
+  { // Sub-select tuning from command line argument
+    const std::vector<std::string>& selected_tuning_names = run_params.getTuningInput();
+    if (selected_tuning_names.size() == 0 || std::find(selected_tuning_names.begin(), selected_tuning_names.end(), name) != selected_tuning_names.end()) {
+      variant_tuning_names[vid].emplace_back(std::move(name));
+    }
+  }
```

Comment:
Do we need to filter here, or is the filtering in Executor.cpp enough to avoid running the tuning?

Reply:
Reverted see [39f9ca9](https://github.com/LLNL/RAJAPerf/pull/333/commits/39f9ca9df0f4785ea100f24a9c106abedff4273d)

~I also found this strange but while testing on Friday it seemed like both were necessary. Empirically I found that only having the `Executor.cpp` filter would target the correct tunings but then still run all of them anyway. Now it seems to be working correctly.~

I just realized that this is probably because I was missing the `selected_tuning_names.empty()` check the first time and now that I have that I don't need the check in `KernelBase.hpp`. Good catch.